modelskill/metrics.py

```diff
@@ -450,6 +450,27 @@ def scatter_index2(obs: np.ndarray, model: np.ndarray) -> float:
     )
 
 
+def ev(obs: np.ndarray, model: np.ndarray) -> float:
+    """EV: Explained variation
+
+    EV is the explained variation and measures the proportion [0 - 1] to which the model accounts for the variation (dispersion) of the observations
+
+   .. math::
+        \\frac{ \\sum_{i=1}^n (obs_i - \\overline{obs})^2 - \\sum_{i=1}^n \\left( (obs_i - \\overline{obs}) - (model_i - \\overline{model}) \\right)^2}{\\sum_{i=1}^n (obs_i - \\overline{obs})^2}
+    
+    Range: [0, 1]; Best: 1
+
```

Comment:
Maybe add a "See Also" with reference to r2?

Reply:
The thing is that I am not sure if it is the same, as r2 can be negative, and this ev cannot.
I do not have the reference for this metric.